content/en/docs/components/pipelines/concepts/pipeline-root.md

```diff
@@ -5,7 +5,7 @@ weight = 15
 
 +++
 
-Starting from [Kubeflow Pipelines SDK v2](https://www.kubeflow.org/docs/components/pipelines/legacy-v1/sdk-v2/) and Kubeflow Pipelines 1.7.0, Kubeflow Pipelines supports a new intermediate artifact repository feature: pipeline root in both [standalone deployment](/docs/components/pipelines/legacy-v1/installation/standalone-deployment/) and [AI Platform Pipelines](https://cloud.google.com/ai-platform/pipelines/docs).
+Starting from [Kubeflow Pipelines SDK v2](https://kubeflow-pipelines.readthedocs.io/en/stable/) and Kubeflow Pipelines 1.7.0, Kubeflow Pipelines supports a new intermediate artifact repository feature: pipeline root in both [standalone deployment](/docs/components/pipelines/legacy-v1/installation/standalone-deployment/) and [AI Platform Pipelines](https://cloud.google.com/ai-platform/pipelines/docs).
```

Comment:
Thank you for fixing the broken link. However, KFP SDK 2.* isn't compatible with Kubeflow Pipelines 1.7 backend. The client in the SDK only works for Kubeflow Pipelines 2.*.

Reply:
@chensun I replaced `Kubeflow Pipelines 1.7.0` with `Kubeflow Pipelines v2`. Please let me know if that's OK.